http/server.go

```diff
@@ -268,6 +268,10 @@ func (s *Server) loadExtraAssets(folder, prefix string) {
 	files := []string{}
 
 	err := filepath.Walk(folder, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
```

Comment:
No return params for this function. look like we can't return the error here, just log the error and return nil is fine.

Reply:
There is already a line that logs this error line 280

Reply:
got it. Thanks!